make/autoconf/build-aux/config.sub

```diff
@@ -46,6 +46,13 @@ if echo $* | grep pc-msys >/dev/null ; then
     exit
 fi
 
+# Canonicalize for riscv64 which autoconf-config.sub doesn't handle
+if echo $* | grep '^riscv64-linux' >/dev/null ; then
+    result=`echo $@ | sed 's/^riscv64-linux/riscv64-unknown-linux/'`
+    echo $result
+    exit
+fi
```

Comment:
Looks like riscv32 has the same problem, we should handle riscv32 here too.

Reply:
Done. Thanks for suggestion!